components/tidb_query/src/codec/batch/lazy_column_vec.rs

```diff
@@ -131,6 +133,48 @@ impl LazyBatchColumnVec {
         Ok(())
     }
 
+    /// Encode into arrow format.
+    pub fn encode_to_chunk(
```

Comment:
Maybe call it `encode_arrow` is better. We have too many "chunk" which have different meanings.